Make adapter and example compatible with preCICE v3.0.0 (#7)
precice/dune-adapter: README.md

## dune-precice
A adapter module enabeling multiphysics simulation inside DUNE with the help of preCICE [1][2].

> The adapter can be built with:

Reviewer: ```suggestion
Build the adapter by running
```

The dune-elastodynamics module can be found here:
https://github.com/maxfirmbach/dune-elastodynamics

> The executable `dune-perpendicular-flap` for the perpendicular-flap tutorial case can be
> found in:

Reviewer: ```suggestion
The executable `dune-perpendicular-flap` for the perpendicular-flap tutorial case is in:
```

`dune-precice-howto/build-cmake/examples/`

> after the module was built and needs to be copied to the `solid-dune` folder inside the respective tutorial case.

Reviewer: This sentence made sense as a standalone bullet point, but does not have a reading flow. What are we trying to say here?
